examples/v1beta1/pipelineruns/no-ci/csi-workspace.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
Any reason why this is in no-ci?

Reply:
This would require the respective csi driver to be installed.
(Later we will come up with another hashicorp example with a detailed guide ideally)